### PR TITLE
Form validation

### PR DESCRIPTION
**Pull request by matthewglover**
FIXES: #44

### CODE REVIEW

File: src/entity/taskslog.js [hunk 1]
```diff
@@ -1,32 +1,30 @@
...
-  const task = nga.entity('taskslog');
+  const taskLog = nga.entity('taskslog');
 
   const fields = [
-    nga.field('tasksName'),
-    nga.field('tenantName'),
-    nga.field('dateAssigned'),
+    nga.field('taskName'),
+    nga.field('tenantName')
+      .isDetailLink(true),
```
Comment (daymos) [inline src/entity/taskslog.js:7]: what does line 7 do?

File: resources/taskslog/config.json [hunk 2]
```diff
@@ -9,61 +9,45 @@
...
+			"order": 4
 		},
 		"dateReplied": {
 			"name": "dateReplied",
 			"type": "number",
 			"typeLabel": "number",
 			"required": false,
 			"id": "dateReplied",
-			"order": 7
+			"order": 5
```
Comment (daymos) [inline resources/taskslog/config.json:50]: i  think order is autogenerated by the dashboard. do you know what it does?

File: resources/messages/post.js [hunk 3]
```diff
@@ -1,37 +1,31 @@
...
 
-dpd.contacts.get({id: {$in:to} })
-  .then(contacts => contacts.map(contact => contact.phoneNumber))
-  .then(phoneNumbers => {
-    phoneNumbers.forEach(phone =>
-      nexmo.message.sendSms(senderNumber, phone, message, {debug:true}, ()=>{}));
+dpd.contacts.get({id: to})
+  .then(contact => {
+    if (apiKey) {
+      nexmo.message.sendSms(SENDER_NUMBER, contact.phoneNumber, 'wtf', { debug:true }, noop);
```
Comment (daymos) [inline resources/messages/post.js:24]: how are you sending the message? Seems you are sending and unpolite standard message here.

File: src/entity/taskslog.js (hunk 1 above)
Comment (matthewglover) [reply src/entity/taskslog.js:7]: Creates a link to the edit view (although I've updated the field which links in the second pull request)

File: resources/taskslog/config.json (hunk 2 above)
Comment (matthewglover) [reply resources/taskslog/config.json:50]: No - I've not actively changed this.

File: resources/messages/post.js (hunk 3 above)
Comment (matthewglover) [reply resources/messages/post.js:24]: True! I've fixed this in the second pull request.

Comment (daymos) [reply resources/messages/post.js:24]: I saw you changed this in the new pull request

**Pull request merged**